cloudformation/stack.yaml

```diff
@@ -1409,8 +1409,9 @@ Resources:
         ZipFile: |
           import json
           import boto3
-          from botocore.vendored import requests
+          import urllib3
           from botocore.exceptions import ClientError
```

Comment:
Are `ClientError`s from `botocore.exceptions` and other `botocore` bits still expected to be used? (Sorry if this is a dumb question -- Is there something I can read to catch up on the `botocore` deprecation issue?)

Reply:
All good. I had this same question and asked an aws support person yesterday, luckily only the botocore.requests library is being deprecated, this one is still functional.